src/containers/text-mode.jsx

```diff
@@ -82,8 +82,9 @@ class TextMode extends React.Component {
         // If fill and stroke color are both mixed/transparent/absent, set fill to default and stroke to transparent.
         // If exactly one of fill or stroke color is set, set the other one to transparent.
         // This way the tool won't draw an invisible state, or be unclear about what will be drawn.
-        const {strokeColor, strokeWidth} = nextProps.colorState;
+        const {strokeWidth} = this.props.colorState;
```

Comment:
This looks like it should be nextProps

Reply:
I filed https://github.com/LLK/scratch-paint/issues/1141 to look into why nextProps is here in the first place